src/metatensor/models/soap_bpnn/model.py

```diff
@@ -4,10 +4,20 @@
 import rascaline.torch
 import torch
 from metatensor.torch import Labels, TensorBlock, TensorMap
+from omegaconf import OmegaConf
+
+from metatensor.models import ARCHITECTURE_CONFIG_PATH
 
 from ..utils.composition import apply_composition_contribution
 
 
+ARCHITECTURE_NAME = "soap_bpnn"
+
+DEFAULT_MODEL_HYPERS = OmegaConf.to_container(
+    OmegaConf.load(ARCHITECTURE_CONFIG_PATH / "soap_bpnn.yaml")
+)["model"]
```

Comment:
I think we don't have to set hypers to a default value or is there a specific reason?

Reply:
In this way, it's much easier to use the model from Python. For example, the test I've written for the model_io module would have been much more difficult. In general, I feel like, if we want to support model use from Python as well, we need to make the hypers available from Python as well

Reply:
Okay I get your point but then maybe it makes sense to provide a general parser and store the hypers as dictionary in the init of each architecture.